canonical/maas-ui
# feat(machines): add group by parent

## Done

- add group by parent
- migrate GroupSelect to RTL
  - remove snapshot test
## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- (using bolla) Go to machine listing page
- Select Group by parent
- Verify the list updates correctly

## Fixes

Fixes: https://github.com/canonical/app-tribe/issues/1416

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in main.

## Screenshots
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/7452681/195596135-12e2a072-3935-4dfe-be08-0b569735b7a7.png">
